src/graph.py

```diff
@@ -581,6 +583,46 @@ def grid_search(self, prize_file, Ws, Bs, Gs):
         return self.grid_randomization(prize_file, Ws, Bs, Gs, 0, 0)
 
 
+    ###########################################################################
+                #######          Summarize randomization results          #######
+    ###########################################################################
+
+    def generate_basic_statistics(self, robust_results): 
+        """
+        Summarizes robust network randomization results
+
+        Arguments: 
+            robust_results (dict of networkx): networkx robust subgraphs indexed by paramstring
```

Comment:
Can you invert this sentence? Maybe just say something like `{paramstring: robust subgraph (networkx)}` to give people a clear picture.